common/culture/innovations/wc_cultural_innovations.txt

```diff
@@ -0,0 +1,220 @@
+#### Local Types ####
```

Comment:
Works better as the cities of gold art IMHO.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/105626119-582d8800-5e47-11eb-98c9-ccfdf9c2ec3d.png)

</details>